bin/tflocal

```diff
@@ -347,6 +363,12 @@ def parse_tf_files() -> dict:
     return result
 
 
+def get_tf_version(env):
+    global TF_VERSION
+    output = subprocess.run(["terraform", "version", "-json"], env=env, check=True, capture_output=True).stdout.decode("utf-8")
```

Comment:
This is a strange legacy code I'm not sure how to handle yet and what was it's real purpose. Tomorrow I try to find it out why we have this in here 🤔

Reply:
After talking with @whummer he explained the difference between the two cases and their purpose so we've figured ideally this must be a subprocess, otherwise the run terminates instantly.